src/thinkspace/client/thinkspace-builder-rat/addon/steps/content.coffee

```diff
@@ -74,15 +87,10 @@ export default step.extend
         @set('assessments', assessments)
         @init_assessments()
 
-  irat_question_items: ember.computed 'irat_assessment.questions_with_answers.@each', ->
-    items = @get('irat_assessment.questions_with_answers')
-    if ember.isPresent(items)
-      @create_question_item(@get('irat_type'), item) for item in items
-
-  trat_question_items: ember.computed 'trat_assessment.questions_with_answers.@each', ->
-    items = @get('trat_assessment.questions_with_answers')
-    if ember.isPresent(items)
-      @create_question_item(@get('trat_type'), item) for item in items
+  # trat_question_items: ember.computed 'trat_assessment.questions_with_answers.@each', ->
```

Comment:
Something that's in the process of being modified in the yet-unpushed `feature/rat-desync` branch, to support re-adding the iRAT/tRAT desync capability